pkg/util/timeutil/time_zone_util_test.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2019 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package timeutil
+
+import "testing"
+
+func TestTimeZoneOffsetStringConversion(t *testing.T) {
+	testCases := []struct {
+		os int64
```

Comment:
nit: can we rename this to `offsetSecs` and `timezone`? these names are not very descriptive :) also prefer if we rearrange the arguments to be {timezone, offsetSecs, ok}
also, thanks for adding negative tests -- can we test that the second argument returns as expected as well?

Reply:
Yes. Absolutely